StreamChatArtifacts.json

```diff
@@ -1 +1 @@
-{}
\ No newline at end of file
+{"4.6.0":"https://github.com/GetStream/stream-chat-swift/releases/download/4.6.0/StreamChat-All.zip"}
```

Comment:
![](https://media.giphy.com/media/1n4FT4KRQkDvK0IO4X/giphy.gif)

Reply:
Love it 😍